pkg/kubernetes/views/topology-page.html

```diff
@@ -61,6 +70,14 @@ <h3 translatable="yes">Node</h3>
       <circle r="15"></circle>
       <text y="7.5">&#xf1b8;</text>
     </g>
+    <g class="DeploymentConfig" id="vertex-DeploymentConfig">
+      <circle r="15"></circle>
+      <text y="8">&#xf013;</text>
+    </g>
+    <g class="Route" id="vertex-Route">
```

Comment:
There is a pficon-route here: https://www.patternfly.org/styles/icons/ Is it available yet in our patternfly?